microsoft-edge/devtools-guide-chromium/whats-new/2021/05/devtools.md

```diff
@@ -0,0 +1,241 @@
+*  Spacing around the tab.
+*  Spacing around the close (`x`) button in the tab.
+*  A background color when hovering over the tab.
+*  A tooltip for the close (`x`) button of the tab.
+*  Higher contrast for the close (`x`) button of the tab.
+
+For example, when you are in the **Animations** tool and you hover over the **Network** tool's tab, the **Network** tab is now formatted to reduce the likelihood of accidentally closing it.
+
+:::row:::
+    :::column:::
+        :::image type="complex" source="../../media/2021/05/hovering-on-tool-tab-in-dark-theme.msft.png" alt-text="When hovering over another tool's tab in dark theme, the tab is formatted to reduce the likelihood of accidentally closing it" lightbox="../../media/2021/05/hovering-on-tool-tab-in-dark-theme.msft.png":::
+           When hovering over another tool's tab in dark theme, the tab is formatted to reduce the likelihood of accidentally closing it.
+        :::image-end:::
+    :::column-end:::
+    :::column:::
+        :::image type="complex" source="../../media/2021/05/hovering-on-tool-tab-in-light-theme.msft.png" alt-text="When hovering over another tool's tab in light theme, the tab is formatted to reduce the likelihood of accidentally closing it" lightbox="../../media/2021/05/hovering-on-tool-tab-in-light-theme.msft.png":::
+           When hovering over another tool's tab in light theme, the tab is formatted to reduce the likelihood of accidentally closing it.
+        :::image-end:::
+    :::column-end:::
+:::row-end:::
+
+These improvements are especially relevant for users of localized DevTools, in which the tabs may be narrower and easier to accidentally close.
+
+
+:::image type="complex" source="../../media/2021/05/hovering-reduced-chance-of-closing-tab.msft.png" alt-text="In localized DevTools, some tabs are narrower, so the reformatting was especially needed" lightbox="../../media/2021/05/hovering-reduced-chance-of-closing-tab.msft.png":::
+   In localized DevTools, some tabs are narrower, so the reformatting was especially needed
+:::image-end:::
+
+We also made it easier to re-add a tool that you closed by adding a [More Tools menu](#add-tools-quickly-with-the-new-more-tools-button) to the main toolbar and drawer toolbar.
+
+
+## Better support for screen readers in the Console
+
+<!-- Title: Better screen reader support in the Console -->
+<!-- Subtitle: Assistive technologies can now announce autocomplete suggestions and evaluated expressions in the Console. -->
+
+In Microsoft Edge version 91 or earlier, in the **Console**, assistive technologies such as screen readers did not announce autocomplete suggestions or the results of evaluated expressions.  In Microsoft Edge version 92, in the **Console**, screen readers now announce the currently selected autocomplete suggestion, and also now announce the result of an evaluated expression.
```

Comment:
Prior to Microsoft Edge version 92, in the **Console**, assistive technologies such as screen readers didn't announce autocomplete suggestions or the results of evaluated expressions. This has been fixed now.